snmp/tests/test_e2e_core_profiles/test_profile_fortinet_fortigate.py

```diff
@@ -0,0 +1,163 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (
+    assert_common_metrics,
+    # assert_extend_fortinet_fortigate_cpu_memory,
+    # assert_extend_fortinet_fortigate_vpn_tunnel,
+    assert_extend_generic_if,
+    create_e2e_core_test_config,
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_fortinet_fortigate(dd_agent_check):
+    config = create_e2e_core_test_config('fortinet-fortigate')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:fortinet-fortigate',
+        'snmp_host:fortinet-fortigate.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+        'device_vendor:fortinet',
+    ] + []
+
+    # --- TEST EXTENDED METRICS ---
+    assert_extend_generic_if(aggregator, common_tags)
+    # assert_extend_fortinet_fortigate_cpu_memory(aggregator, common_tags)
+    # assert_extend_fortinet_fortigate_vpn_tunnel(aggregator, common_tags)
```

Comment:
are these going to be addressed in a separate PR ?
if that's the case can you remove the comments and include them when the changes are made ?
if tests are missing we don't keep the comments